lib/src/widgets/switch/ds_switch_base.widget.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+abstract class DSSwitchBase extends StatelessWidget {
+  const DSSwitchBase({
+    Key? key,
+    required this.onChanged,
+    required this.isActive,
```

Comment:
It would be nice if we use the same pattern Material Switch use, rather than "isActive, we should use "value", what do you think?